yb-voyager/src/query/queryparser/helpers_struct.go

```diff
@@ -58,22 +39,6 @@ func IsMviewObject(parseTree *pg_query.ParseResult) bool {
 	return isCreateAsStmt && createAsNode.CreateTableAsStmt.Objtype == pg_query.ObjectType_OBJECT_MATVIEW
 }
 
-func GetSelectStmtQueryFromViewOrMView(parseTree *pg_query.ParseResult) (string, error) {
```

Comment:
Ah, so we don't really need these anymore because parser can directly parse MVIEW/VIEW, correct?

Reply:
yeah